src/chromium/interfaces/getters.js

```diff
@@ -1,19 +1,29 @@
 import { getExternalBrowser } from "Shared/backgroundScripts/getters.js";
 
+const nativeApps = [
+  "org.mozilla.firefox_bridge_nmh_dev",
```

Comment:
Let's do this order:
- release
- nightly
- esr
- dev

Then the most commonly installed flavor is found first.

Reply:
In the attached document, under "Different Firefox channel", it says to use this order. I believe it's because users who have two or more releases, they likely use the more "dev-ier" one.

Reply:
> Let's do this order:
> 
>     * release
> 
>     * nightly
> 
>     * esr
> 
>     * dev
> 
> 
> Then the most commonly installed flavor is found first.

My thinking was based on user deliberate actions. A developer is most likely to use and test a dev build they did locally. Someone using nightly is doing so deliberately regardless of if they have a release build installed. And we should prefer newer versions of everything over ESR if a newer version is installed.

If the user wants to use release, they can likely more easily delete the dev or nightly build, if they use multiple channels at once. If they don't use multiple channels, it doesn't matter.

Rob, do you have any data to indicate users are more likely to want the release build channel to be the first one that should be connected with?

This should really be user configurable in some way. For the next iteration, we can add a configurable setting in the Chromium extension. For now, any order (including the one Greg has already done) should be fine to commit, as long as we communicate it to users when we make this available.